custom_components/adaptive_lighting/switch.py

```diff
@@ -1254,13 +1255,12 @@ def get_settings(
             rgb_color: tuple[float, float, float] = color_temperature_to_rgb(
                 color_temp_kelvin
             )
-        color_temp_mired: float = color_temperature_kelvin_to_mired(color_temp_kelvin)
         xy_color: tuple[float, float] = color_RGB_to_xy(*rgb_color)
         hs_color: tuple[float, float] = color_xy_to_hs(*xy_color)
         return {
             "brightness_pct": brightness_pct,
             "color_temp_kelvin": color_temp_kelvin,
-            "color_temp_mired": color_temp_mired,
+            "color_temp_kelvin": color_temp_kelvin,
```

Comment:
Looks like this duplicates the previous line

Reply:
Thanks!